source/text/Format.h

```diff
@@ -28,6 +28,9 @@ class Format {
 	static std::string Credits(int64_t value);
 	// Convert a time in seconds to years/days/hours/minutes/seconds
 	static std::string PlayTime(double timeVal);
+	//Abbreviates numbers in the tens of thousands with the suffix "k"
+	//to conserve space in the box for secondary ammo
```

Comment:
```suggestion
	// Convert the given number into a string, abbreviating numbers in the tens of thousands
	//  with the suffix "k", to conserve space in the box for secondary ammo.
```
Lets match the other comments in style.
Then you should add it in front of the function in the cpp, to match the other functions there.